controllers/drpolicy_controller.go

```diff
@@ -243,6 +250,36 @@ func s3ProfileValidate(ctx context.Context, apiReader client.Reader,
 	return "", nil
 }
 
+func validateCIDRsFormat(ctx context.Context, drpolicy *ramen.DRPolicy, log logr.Logger) (string, error) {
+	// validate the CIDRs format
+	for i := range drpolicy.Spec.DRClusterSet {
+		cluster := &drpolicy.Spec.DRClusterSet[i]
+		if reason, err := CIDRsFomatValidate(ctx, cluster.CIDRs, log); err != nil {
+			return reason, err
+		}
+	}
+
+	return "", nil
+}
+
+func CIDRsFomatValidate(ctx context.Context, cidrs []string, log logr.Logger) (string, error) {
```

Comment:
no need to have this extra exported function move the logic to `validateCIDRsFormat` function?

Reply:
I initially had written with one function. then I checked the `validateS3Profiles` function [here](https://github.com/RamenDR/ramen/blob/main/controllers/drpolicy_controller.go#L217-L230). It was recently split to make it look clean I believe. I can still make the change. What do you think?

Reply:
should be fine as the pattern is already available.